kotlin/src/test/TestApiSettings.kt

```diff
@@ -36,41 +71,61 @@ class TestApiSettings {
     fun testApiSettingsDefaults() {
         val settings = ApiSettings.fromIniText(bareMinimum)
         val config = settings.readConfig()
-        assertEquals(settings.baseUrl, "https://my.looker.com:19999", "Base URL is read")
-        assertEquals(settings.verifySSL, true)
-        assertEquals(settings.timeout, DEFAULT_TIMEOUT)
-        assertEquals(config["client_id"], null)
-        assertEquals(config["client_secret"], null)
+        assertEquals("https://my.looker.com:19999", settings.baseUrl, "Base URL is read")
+        assertEquals(ENVIRONMENT_PREFIX, settings.environmentPrefix)
+        assertEquals(true, settings.verifySSL)
+        assertEquals(DEFAULT_TIMEOUT, settings.timeout)
+        assertNull(config["client_id"])
+        assertNull(config["client_secret"])
     }
 
     @Test
     fun testApiSettingsQuotes() {
         val settings = ApiSettings.fromIniText(quotedMinimum)
-        assertEquals(settings.baseUrl, "https://my.looker.com:19999", "Base URL has no quotes")
-        assertEquals(settings.verifySSL, true)
-        assertEquals(settings.timeout, DEFAULT_TIMEOUT)
+        assertEquals("https://my.looker.com:19999", settings.baseUrl, "Base URL is read")
+        assertEquals(ENVIRONMENT_PREFIX, settings.environmentPrefix)
+        assertEquals(true, settings.verifySSL)
+        assertEquals(DEFAULT_TIMEOUT, settings.timeout)
     }
 
     @Test
     fun testApiSettingsOverrides() {
         val settings = MockSettings(bareMinimum)
+        assertEquals("https://my.looker.com:19999", settings.baseUrl, "Base URL is read")
+        // TODO is there a way the following line of code would work without reassigning settings from the map of readConfig()
+        // assertEquals(mySDK, settings.environmentPrefix)
```

Comment:
We should figure out a way the overridden `readConfig()` is used to initialize the `settings` values assigned here. This doesn't match the inheritable behavior of ApiSettings for the other SDKs, and will confuse anyone who tries to create their own override.